docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -65,15 +49,21 @@ Upgrade Assistant provides information about which {dfeeds} need to be updated.
 ===============================
 --
 
-. {ref}/docs-reindex.html[Reindex] or delete any indices created on 2.n. We recommend
-upgrading to the most recent 5.6 and using the {xpack} Reindex Helper to reindex 2.n indices.
-
-. If Kibana and {xpack} are part of your stack, upgrade the internal Kibana
-and {xpack} indices. We recommend using the {xpack} 5.6 Reindex Helper to
-upgrade the internal indices. If you're performing a full cluster restart upgrade
-from an earlier version, you can also <<upgrade-internal-indices,use the
- `_xpack/migration/upgrade` API>> directly to upgrade the
-internal indices after you install Elasticsearch {version}.
+. Upgrade to the 5.6 or later and use the
+{kibana-ref}/upgrade-assistant.html[{kib} Upgrade Assistant] to
+{ref}/docs-reindex.html[reindex] any indices that are not compatible with {version}.
++
+[role="xpack"]
+.Upgrade Assistant
+******
+The Upgrade Assistant and migration APIs are enabled with both the Basic and
+Trial licenses. You can install the default distribution of 6.3 to use the
+Upgrade Assistant to prepare to upgrade even if you are upgrading to the OSS
```

Comment:
This reference to 6.3 is incorrect. You don't need to install 6.3 to get to 6.7. :-)

Reply:
Right, but the default distribution doesn't exist until 6.3. Let's see if I can make that clearer.